submitted_models/marble_spot_sensor_config_1/model.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<model>
+  <name>ctu_cras_norlab_spot_sensor_config_1</name>
+  <sdf version="1.6">model.sdf</sdf>
+  <description>
+    Boston Dynamics Spot of CTU-CRAS-Norlab team.
+  </description>
```

Comment:
Should this be Marble as opposed to CTU-CRAS-Norlab?

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d